src/lib.rs

```diff
@@ -364,6 +368,126 @@ pub fn main() {
                 }
             }
         }
+        Commands::Merge {
+            ref_db1,
+            ref_db2,
+            output,
+        } => {
+            
+            let ref_db_name1 = MultiSketch::strip_sketch_extension(ref_db1);
+            let ref_db_name2 = MultiSketch::strip_sketch_extension(ref_db2);
+
+            let mut sketches1: MultiSketch = MultiSketch::load(ref_db_name1).unwrap_or_else(|_| {
+                panic!("Could not read sketch metadata from {ref_db_name1}.skm")
+            });
+            let mut sketches2: MultiSketch = MultiSketch::load(ref_db_name2).unwrap_or_else(|_| {
+                panic!("Could not read sketch metadata from {ref_db_name2}.skm")
+            });
+
+            // make &str
+            let str_output: &str = output.as_ref().unwrap().as_str();
+            // read in the two db 
+            log::info!("Loading sketch data from {}.skd", ref_db_name1);
+            sketches1.read_sketch_data(ref_db_name1);
+            log::info!("Loading sketch data from {}.skd", ref_db_name2);
+            sketches2.read_sketch_data(ref_db_name2);
+
+            // check compatibility
+            if !&sketches1.is_compatible_with(&sketches2) {
+                panic!("Databases are not compatible for merging.")
+            }
+
+            let merged_sketch = &sketches1.merge_sketches(&sketches2);
+
+            let _ = MultiSketch::save_metadata(merged_sketch, str_output);
```

Comment:
With these functions which return a result, it's best to write them like this:
```
MultiSketch::save_metadata(merged_sketch, str_output).expect("Couldn't save metadata to {str_output}");
```
Which checks they didn't error, and quits and prints a message if they did.